e2e/app/component/data-resource-card.ts

```diff
@@ -84,6 +84,14 @@ export default class DataResourceCard extends CardBase {
     return null;
   }
 
+  async findAnyCard(cardType: ResourceCard): Promise<DataResourceCard | null> {
```

Comment:
A new helper function that will help with test code efficiency. Right now it's not used by any test.